install.bat

```diff
@@ -0,0 +1,50 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0."
+
+ECHO "Cloning the repository"
+git clone https://github.com/systers/mentorship-backend.git
```

Comment:
Can you please make this optional? Maybe asking the user before the entire process begins would be great since he/she would usually already have a clone to be able to have this file, but sometimes he/she may want a clean setup!